tests/test_scripts.py

```diff
@@ -689,6 +689,21 @@ def test_oxli_build_graph_fail():
     assert "** ERROR: the graph structure is too small" in err
 
 
+@pytest.mark.known_failing
+def test_oxli_build_graph_yuge():
+    script = 'oxli'
+    args = ['build-graph', '-M', '800T', '-k', '20']
```

Comment:
Like all of the rest of the `load-graph.py` tests, this is marked "known failing" since loading the oxli library for pytest seems like an unsolved problem.